modules/simple_forms_api/app/models/simple_forms_api/vba_21_0966.rb

```diff
@@ -93,6 +99,82 @@ def track_user_identity(confirmation_number)
 
     private
 
+    def veteran_ssn
+      [
+        data.dig('veteran_id', 'ssn')&.[](0..2),
+        data.dig('veteran_id', 'ssn')&.[](3..4),
+        data.dig('veteran_id', 'ssn')&.[](5..8)
```

Comment:
do we also want to check for 'veteran_id', 'va_file_number' ?

Reply:
This field isn't split up the way the ssn field is so it'll get caught in our regular sweep for user input. The changes in this PR are only for fields that are split up (and therefore don't get caught when we just remove user-input directly).